lib/picker.js

```diff
@@ -23,8 +23,7 @@
 var $window = $( window )
 var $document = $( document )
 var $html = $( document.documentElement )
-var supportsTransitions = document.body.style.transition != null
-
+var supportsTransitions = function() { return document.body.style.transition != null; }
```

Comment:
I tried to build with grunt, but one of the tests are now not passing.
```
Date picker `set` - `disable` and `enable` using relative ranges
```
This does not seem to be affected by any of my changes.